File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1052.go
package main
 
import (
    "fmt"
)

func main() {
    var n int;

    fmt.Scanf("%d", &n);

    meses := [12]string{"January", "February", "March", "April", "May", "June", "July", "August", "September", "October", "November", "December"};        

    fmt.Println(meses[n-1]);

}

File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1052.c
#include<stdio.h>

int main() {
    int n;
    scanf("%d", &n);
    
    char meses[30][12] = {"January", "February", "March", "April", "May", "June", "July", "August", "September", "October", "November", "December"};
    
    printf("%s\n", meses[n-1]);
    
    return 0;
}